src/piltover_input.cairo

```diff
@@ -0,0 +1,77 @@
+use piltover::snos_output::{StarknetOsOutput, deserialize_os_output};
+
+pub const HEADER_SIZE: usize = 8;
+
+#[derive(Drop, Serde, Debug)]
+pub struct LayoutBridgeOutput {
```

Comment:
Some comments for the reader about the layout bridge output that now contains the SNOS output etc..